assets/css/frontend.less

```diff
@@ -140,6 +140,17 @@
 			margin: 0;
 			align-items: center;
 		}
+
+		&__field{
+			&--text{
+				.wp-picker-clear, .wp-color-result-text{
+					line-height: 40px !important;
```

Comment:
Is this some issue specific to the front-end styling, caused by HiveTheme styles, or these styles are not loaded on the front-end but added by WordPress on back-end or Customizer?

Reply:
Yes, there is issue with the clear button in the color picker. It gets HiveTheme styles and shows incorrectly. So it is needed to add these styles for properly displaying of the color field